src/scenarios/blazor/test.py

```diff
@@ -0,0 +1,13 @@
+'''
+Empty Blazor Wasm Template
+'''
+from shared.runner import TestTraits, Runner
+
+SCENARIONAME = 'Empty Blazor Wasm Template'
```

Comment:
```suggestion

```

Reply:
oops, missed this comment. I should probably remove it from all ```test.py```